v2/cf-nodejs-logging-support/src/test/acceptance-test/custom-fields.test.js

```diff
@@ -64,7 +64,7 @@ describe('Test custom fields', function () {
             });
         });
 
-        describe('application-logs format', function () {
+        describe.skip('application-logging format', function () {
```

Comment:
We cannot skip this test the format needs to be verified. Using `setCustomFieldsFormat` would be fine as long as detecting "application-logs" and "cloud-logging" service binding and correct format setting gets checked elsewhere. I couldn't find such a test.

It is a requirement to support following custom field formats:
* `cloud-logging` => done
* `application-logging` => done, but please use "application-logging" as format name. However, detecting it in bound services is still done by checking for "application-logs".
* `all` => use application-logging and cloud-logging format in parallel. This should be the case if both services are bound.
* `disabled` => do not log any custom fields
* `default` => same as `cloud-logging`

I would recommend the exact names from above list to keep track of the currently set custom fields format. This allows setting it in a consistent way via API and config. The service binding detection needs to be updated accordingly.

Since we want to switch from `overrideCustomFieldFormat ` to `setCustomFieldsFormat` API, we have to add a description to the migration guide (part of release notes) to make sure people replace `overrideCustomFieldFormat` with `setCustomFieldsFormat`. See existing docs https://sap.github.io/cf-nodejs-logging-support/general-usage/custom-fields#general-use